entries/c/cineplex.com.json

```
@@ -0,0 +1,16 @@
+{
+  "Cineplex": {
+    "domain": "cineplex.com",
+    "tfa": [
+      "call",
+      "sms"
+    ],
+    "documentation": "https://www.cineplex.com/policy/privacy-policy",
```

Whilst the document you linked to does mention 2FA, it's not clear how 2FA is enabled and used. I recommend this page instead:
```suggestion
    "documentation": "https://cineplex.service-now.com/gsc?id=kb_article&sysparm_article=KB0011023",
```